### ISSUE TITLE
GPU Memory required for WaLa

### ISSUE THREAD

**Issue by supersyz** [AutodeskAILab/WaLa#11]
Really appreciate for your great job！ Would you mind tell me the requirement of the hardware?

**Comment by hoosha** [comment]
You are right. We should clarify this in the readme. In the mean time, the Colab demo statement might help you: "your need a colab runtime with at least 20GM system RAM and 16GB GRAM."
@kamal-rahimi @MartynasPocius
